app/views/orga/conferences/_form.html.erb

```diff
@@ -1,14 +1,18 @@
 <h1>Import conferences (.csv only)</h1>
 <p>Importing will update existing conferences with the data from the csv file</p>
 
-<%= form_tag import_orga_conferences_path, multipart: true, class: "form-inline" do %>
-  <%= file_field_tag :file, accept: '.csv' %>
-  <%= submit_tag "Import csv file", class: "btn btn-default btn-small"  %>
+<%= form_tag import_orga_conferences_path, multipart: true do %>
+  <div class="form-group">
+    <%= file_field_tag :file, accept: '.csv', id: 'input-csv-file-upload' %>
```

Comment:
Either way, how about setting `autofocus: true` on the file field? That makes is hard to miss.